infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/workflows/AddNode.java

```diff
@@ -0,0 +1,79 @@
+package org.corfudb.infrastructure.orchestrator.workflows;
+
+import lombok.Getter;
+import org.corfudb.infrastructure.orchestrator.Action;
+import org.corfudb.infrastructure.orchestrator.Workflow;
+import org.corfudb.runtime.CorfuRuntime;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * A workflow that adds a new node to the cluster.
+ *
```

Comment:
When is it used? How does it add the node?

Reply:
Have a dependency on another PR. Will fill in this stub later.